salt/states/solrcloud.py

```diff
@@ -162,15 +162,15 @@ def collection(name, options=None, **kwargs):
             ret['comment'] = 'The collection "{0}" will be created.'.format(name)
             ret['pchanges'] = {
                 'old': None,
-                'new': str('options=') + new_changes  # future lint: disable=blacklisted-function
+                'new': six.text_type('options=') + new_changes  # future lint: disable=blacklisted-function
```

Comment:
This should be reverted, we're concatenating to the result of a json.dumps, which is a `str()` type.